exec/java-exec/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -241,8 +243,8 @@ SqlNode SqlCreateTable() :
     <AS>
     query = OrderedQueryOrExpr(ExprContext.ACCEPT_QUERY)
     {
-        return new SqlCreateTable(pos,tblName, fieldList, partitionFieldList, query,
-                                    SqlLiteral.createBoolean(isTemporary, getPos()));
+        return new SqlCreateTable(pos, SqlLiteral.createBoolean(tableNonExistenceCheck, getPos()), tblName, fieldList,
```

Comment:
Not sure if it is a good practice to add new parameter in the beginning of constructor rather in the end.